x-pack/plugins/observability_solution/observability_ai_assistant/server/service/util/get_system_message_from_instructions.ts

```diff
@@ -40,29 +40,29 @@ export function getSystemMessageFromInstructions({
 
   const requestInstructionsWithId = requestInstructions.map((instruction) =>
     typeof instruction === 'string'
-      ? { doc_id: v4(), text: instruction, system: false }
+      ? { doc_id: v4(), text: instruction, user_instruction: true }
```

Comment:
I flipped the logic around from `system` to `user_instruction`. "system" doesn't make sense to me since they are all system messages. What we want to differentiate on is whether they are user instructions or "registeret" (built-in) instructions.

Reply:
What happens with the instructions to the /chat/complete endpoint? I don't see anything changed there.

Reply:
Addressed in 7c65ee900b26eb21ebc4c571286cdb28532fbe08